mod-kitpvp/src/main/java/gg/solarmc/loader/kitpvp/OnlineKitPvp.java

```diff
@@ -50,6 +56,21 @@ void updateAssists(int i) {
         this.assists = i;
     }
 
+    @Override
+    void updateExperience(int i) {
+        this.experience = i;
+    }
+
+    @Override
+    void updateCurrentKillstreak(int i) {
+        this.currentKillstreaks = i;
+    }
+
+    @Override
+    void updateHighestKillstreak(int i) {
+        this.highestKillstreaks = i;
+    }
+
```

Comment:
Since you're caching these values, I'm assuming you will add methods like `public int currentHighestKillstreak()`

Reply:
corrent, i wrote this pull request at 1 am and my brain forgot to add cached value methods